### ISSUE TITLE
i18n support

### ISSUE THREAD

**Issue by mirkoAlic** [emeks-studio/sensum-mobile#5]
## Description

It would be really cool at least to support for english and spanish (or kind of hehe)
- suggested lib: https://github.com/i18next/react-i18next

**Comment by randomeduc** [comment]
Only the UI messages but not the sensations text, right? :thinking:

**Comment by mirko-plowtech** [comment]
> Only the UI messages but not the sensations text, right? thinking

Right. I mean, sensation texts will not be translated (Obs: users can write "any" text)

**Comment by mk-emeks** [comment]
for the pwa: https://tech.ahrefs.com/localization-system-for-reason-rescript-react-apps-6bf03aa1fdec
